# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-text/white-space/text-wrap-balance-line-clamp-001.html.ini
# Commit: 8c9ba927121c
# Full Hash: 8c9ba927121cd7f1cb5b31ffb86addc4c34b59a8
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2023-09-30 21:19:54
# Regressor Bug: 1731541
# File Overlap Count: 1
# Description:
#   Bug 1731541 - When line-clamp is in effect, make text-wrap:balance consider only the lines up to the clamp limit. r=emilio
#   
#   This corresponds to how Chrome behaves, and passes the test they included in WPT.
#   
#   It's unclear to me whether this behavior actually follows from the current spec
# ==============================================================================

diff -r 77fce08f9637 -r 8c9ba927121c testing/web-platform/meta/css/css-text/white-space/text-wrap-balance-line-clamp-001.html.ini
--- a/testing/web-platform/meta/css/css-text/white-space/text-wrap-balance-line-clamp-001.html.ini	Sat Sep 30 15:53:12 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,2 +0,0 @@
-[text-wrap-balance-line-clamp-001.html]
-  expected: FAIL
